public/assets/sass/pages/resources/cheque-requests.scss

```
@@ -0,0 +1,74 @@
+@import "../../util/_sass-mixins";
+@import "../../util/variables";
+@import "../../mixins/banner-image";
+
+section {
+    max-width: var(--content-width);
+    margin: 30px auto;
+    position: relative;
+}
+
+.collapsible-button {
+    background-color: #fff;
+    color: #444;
+    cursor: pointer;
+    padding: 18px;
+    width: 100%;
+    border: none;
+    text-align: left;
+    outline: none;
+    font-size: 20px;
+    border-top: 2px solid;
+
+    ::after {
+        content: '\02795'; /* Unicode character for "plus" sign (+) */
+        font-size: 13px;
+        color: white;
+        float: right;
+        margin-left: 5px;
+    }
+
+    .active:after {
+        content: "\2796"; /* Unicode character for "minus" sign (-) */
+    }
+}
+
+.collapsible-content {
+    padding: 0 18px;
+    background-color: white;
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.2s ease-out;
+}
+  
+.collapsible-button:after {
+    content: '\02795'; /* Unicode character for "plus" sign (+) */
+    font-size: 13px;
+    color: white;
+    float: right;
+    margin-left: 5px;
+}
+
+.active:after {
+    content: "\2796"; /* Unicode character for "minus" sign (-) */
+}

```

Rather than copying and pasting the styling for the collapsible component, we could instead import it from the existing `_collapsible.scss` file :eyes: 

```suggestion
@import "../../mixins/collapsible";
```

---

Good catch, done!